examples_helper_test.go

```diff
@@ -24,13 +25,13 @@ type failing struct{}
 func (f *failing) Fail() {}
 
 // documentation helper just for the example
-func printFileContent(path string) {
-	approvedPath := strings.Replace(path, ".received.", ".approved.", 1)
-	content, err := ioutil.ReadFile(approvedPath)
+func printFileContent(filepath string) {
+	approvedPath := strings.Replace(filepath, ".received.", ".approved.", 1)
+	content, err := ioutil.ReadFile(path.Join("testdata", approvedPath))
```

Comment:
I suspect this hardcoded "testdata" should refer instead to approvals.defaultFolder ?

Reply:
it is actually a bit more complicated because this file is in the approvals_test package, so it should not have access to defaultFoler.  So I will revert these changes,